binding_generator.py

```diff
@@ -92,10 +92,10 @@ def generate_virtual_version(argcount, const=False, returns=False):
 		return false;\\
 	}\\
 	_FORCE_INLINE_ bool _gdvirtual_##m_name##_overridden() const {\\
-		return godot::internal::gdextension_interface_object_has_script_method(_owner, &_gdvirtual_##m_name##_sn); \\
+		return ::godot::internal::gdextension_interface_object_has_script_method(_owner, &_gdvirtual_##m_name##_sn); \\
 	}\\
 	_FORCE_INLINE_ static MethodInfo _gdvirtual_##m_name##_get_method_info() {\\
```

Comment:
I just tested this PR. Missing a `::godot::` here

`_FORCE_INLINE_ static ::godot::MethodInfo _gdvirtual_##m_name##_get_method_info() {\\`